packages/ra-core/package.json

```diff
@@ -42,6 +42,7 @@
     "dependencies": {
         "babel-runtime": "~6.26.0",
         "classnames": "~2.2.5",
+        "date-fns": "^1.29.0",
```

Comment:
By adding this version of `date-fns`, you'll import the **whole** library to the react-admin's bundle.

The pre-release `date-fns@2.0.0-alpha.22` supports tree shaking and expose ES modules. And it's stable enough to be imported in React Admin IMHO.

See comparison between:
- [date-fns@1.29.0](https://bundlephobia.com/result?p=date-fns@1.29.0)
- [date-fns@2.0.0-alpha.22](https://bundlephobia.com/result?p=date-fns@2.0.0-alpha.22)

Reply:
I'm not comfortable adding an alpha dependency at this point, especially since the benefit is only in development...

Reply:
ping @Kmaschta

Reply:
It's ok, if it's only used on dev!